src/picongpu/include/simulation_defines/unitless/speciesAttributes.unitless

```diff
@@ -109,7 +109,7 @@ struct Unit<globalCellIdx<T_Type> >
 };
 
 template<>
-struct Unit<chargeState >
+struct Unit<boundElectrons>
 {
   static std::vector<double> get()
   {
```

Comment:
same question as with the `radiationFlag`: seems to be an empty vector... -> needs verification